Api/Startup.cs

```diff
@@ -82,9 +89,7 @@ public void Configure(IApplicationBuilder app)
                 app.Use(async (context, next) =>
                 {
                     var request = context.Request;
-                    request.Headers.Add(
-                        "X-Real-IP", 
-                        "0.0.0.0");
+                    request.Headers["X-Real-IP"] = "0.0.0.0";
```

Comment:
I changed this since it gave a build warning. Probably new analyzers being helpful, but it's still nice to clear out warnings.